dashboard/test/lib/services/afe_enrollment_test.rb

```diff
@@ -32,6 +32,8 @@ def setup
           'csta-plus' => '1',
           'aws-educate' => '1',
           'amazon-terms' => '1',
+          'primary-professional-role' => 'test role with space',
+          'grades-teaching' => 'K-5',
```

Comment:
What does this look like when there are multiple grades selected?

Reply:
It parses kinda funny, but it looks like, 'K-5, 6-8, ', with a trailing ", ". I'll add a test case to cover this.

Reply:
oops I'm sorry, just realized that was what you were talking about in the description 🤦‍♀️ thanks for adding the tests!